src/constants.ts

```diff
@@ -88,7 +88,15 @@ export const HANDLE_SANITIZE_REGEX = /[^\d .A-Za-z]/g;
 // Utils
 export const ALLOWED_IMAGE_TYPES = ['image/jpeg', 'image/png', 'image/webp', 'image/gif'];
 export const ALLOWED_AUDIO_TYPES = ['audio/wav', 'audio/mpeg', 'audio/ogg', 'audio/mp4'];
-export const ALLOWED_MEDIA_TYPES = ['video/mp4', ...ALLOWED_IMAGE_TYPES, ...ALLOWED_AUDIO_TYPES];
+export const ALLOWED_MEDIA_TYPES = [
+  'video/mp4',
+  'video/mpeg',
+  'video/ogg',
+  'video/webm',
+  'video/quicktime',
```

Comment:
I think having these as separate variable and then re using the variable like it has been done for other types would be better? We might need these allowed video types later on?

Reply:
yep, i added those for audio, image. will do the same for video types. thanks